scaleway/resource_lb_frontend_beta.go

```diff
@@ -2,6 +2,11 @@ package scaleway
 
 import (
 	"math"
+	"sort"
```

Comment:
Please use goimports formater

Reply:
uhm I am using it. 
```
 goimports resource_lb_frontend_beta.go | head -15
package scaleway

import (
        "math"
        "sort"

        "github.com/google/go-cmp/cmp"

        "github.com/scaleway/scaleway-sdk-go/scw"

        "github.com/hashicorp/terraform-plugin-sdk/helper/schema"
        "github.com/hashicorp/terraform-plugin-sdk/helper/validation"
        "github.com/scaleway/scaleway-sdk-go/api/lb/v1"
)
```

Reply:
Edit: actually turns out for some reason goimports was not removing empty lines, after doing so by hand it keeps them united. Can be closed